corehq/apps/userreports/reports/README.md

```diff
@@ -0,0 +1,58 @@
+Reports Overview
+================
+
+There are two main components involved in producing a report:
+
+* the data source configuration
+* the report configuration
+
+
+The data source
+---------------
+
+The data source is usually a table in PostgreSQL, but data sources can also be ElasticSearch indexes.
+
+A data source is defined in a DataSourceConfiguration instance. This determines where its data comes from (cases or forms), and how the data is filtered. The data source is populated by a background task. Data sources are not aggregated.
```

Comment:
> Data sources are not aggregated.

Perhaps it should say that there are generally 1 (sometimes, but rarely more) row per case/form?